### PR TITLE
Reliability improvement to startup/shutdown script

### PR DESCRIPTION
**Pull request by austinchambers**

### CODE REVIEW

File: customer/autostartup/shutdown.sh [hunk 1]
```diff
@@ -7,10 +7,11 @@ kubectl config use-context $KUBE_CONTEXT
 
 echo "Stopping Replicated"
 REPLICATED_POD_ID=$(kubectl get pod -l "app=replicated,tier=master" -o name | sed 's/pod\///')
-kubectl exec $REPLICATED_POD_ID -c replicated -- replicatedctl app stop --force
+kubectl exec $REPLICATED_POD_ID -c replicated -- replicatedctl app stop --force --attach
 
+# Prevents issues on shutdown due to fs-snapshotter deployment
 echo 'Delete replicated-shared-fs-snapshotter deployment'
-kubectl delete deployments/replicated-shared-fs-snapshotter
+kubectl delete deployments/replicated-shared-fs-snapshotter || true
```
Comment (dexhorthy) [inline customer/autostartup/shutdown.sh]: can we add a comment about why this is okay to ignore

File: customer/autostartup/startup.sh [hunk 2]
```diff
@@ -6,34 +6,58 @@ echo "Connecting to Cluster"
...
+    echo "Aborting... Unable to query ceph after $ATTEMPTS attempts."
+    echo exit 1
+fi
+
+echo "Waiting for Ceph to be Ready"
+CEPH_HEALTH=$($HEALTH_CMD)
 while [ "$CEPH_HEALTH" != "HEALTH_OK" ]
 do 
-    CEPH_HEALTH=$(kubectl exec -n rook-ceph -i --namespace rook-ceph-system $(kubectl get -n rook-ceph-system pod -l "app=rook-ceph-operator" -o jsonpath='{.items[0].metadata.name}') -- ceph health)
+    CEPH_HEALTH=$HEALTH_CMD)
```
Comment (dexhorthy) [inline customer/autostartup/startup.sh]: extra paren?

Comment (austinchambers) [reply customer/autostartup/startup.sh]: Fixed

File: customer/autostartup/shutdown.sh (hunk 1 above)
Comment (austinchambers) [reply customer/autostartup/shutdown.sh]: Done

**Pull request merged**